data_hub_api/docmaps/v2/sql/docmaps_index.sql

```diff
@@ -304,6 +304,19 @@ t_preprint_published_at_date_and_tdm_path AS (
     AND CAST(tdm.tdm_ms_version AS STRING) = result.preprint_version
 ),
 
+t_manuscript_published_date AS (
+  SELECT
+    * EXCEPT(rn)
+  FROM
+  (
+    SELECT 
+      *,
+      ROW_NUMBER() OVER(PARTITION BY elife_doi, elife_doi_version ORDER BY imported_timestamp DESC) AS rn
+    FROM `elife-data-pipeline.prod.manuscript_published_date`
```

Comment:
It is something I should have noticed when reviewing the PR to add the config...
Maybe `manuscript_published_date` is a bit too generic?
We do already have `published_dates` coming from Observer.

Whereas maybe the scope for this is perhaps more narrow. They are manually provided publication dates for reviewed preprints.
(Not sure what a good name would be)